packages/flutter_test/lib/src/goldens.dart

```diff
@@ -53,8 +53,15 @@ abstract class GoldenFileComparator {
   /// Compares the pixels of decoded png [imageBytes] against the golden file
   /// identified by [golden].
   ///
-  /// The returned future completes with a boolean value that indicates whether
-  /// the pixels decoded from [imageBytes] match the golden file's pixels.
+  /// If [isFlaky] is false, the returned future completes with a boolean value
+  /// that indicates whether the pixels decoded from [imageBytes] match the
+  /// golden file's pixels.
+  ///
+  /// If [isFlaky] is true, the comparison always passes. Implementations should
+  /// strive to assist developers with managing flaky golden tests effectively.
+  /// For example, the comparator could submit the screenshot to the goldens
```

Comment:
We're already agnostic and un-opinionated about this as this doc comment points out, which makes me think we should preserve that.